adserver/templates/adserver/advertiser/flight-list.html

```diff
@@ -18,6 +18,16 @@
 <h1>{% block heading %}{% trans "Manage advertising flights" %}{% endblock heading %}</h1>
 
 
+<aside class="col text-right">
+  {% if request.user.is_staff %}
```

Comment:
Should this use the video recording permission, so users don't get confused and want it?

Reply:
Probably just check "flight create" permissions more precisely.